### DISCUSSION TITLE
Does pantable support footnote?

### DISCUSSION THREAD

**Discussion by gepcel** [ickc/pantable#35] [Q&A] [answered]
Hi, I like pantable, and use it a lot. But I can't get footnotes done.

Here is my example:

testfootnote.md
````
# First paragraph

Some text first[^note1].

```table
---
caption: Caption of table[^note2]
markdown: true
---
Col1, Col2
A, B
```
[^note1]: Some description here.

[^note2]: Description for table.
````

Convert it to pdf using `pandoc testfootnote.md -t latex --filter=pantable -o test.pdf`.
`note1` works, but `note2` doesn't. Any way I can may make this work?

Thanks

**Comment by gepcel** [comment]
It turns out that `inline_notes` works. For example:

````
```table
---
caption: Caption of table^[Description for table]
markdown: true
---
Col1, Col2
A, B
```
````

So I put the solution here, and close this issue.

**Comment by ickc** [comment]
Reopen to ensure I check this behavior. Probably no solution but at least document it.

**Comment by ickc** [comment, marked as answer]
In addition to the inline footnote you used, you could do it like this:

~~~markdown
# First paragraph

Some text first[^note1].

```table
---
caption: |
    Caption of table[^note2]

    [^note2]: Description for table.
markdown: true
---
Col1, Col2
A, B


```
[^note1]: Some description here.
~~~

Essentially the entirety of the value of `caption` will be processed as an individual markdown fragment in terms of function calls.

Another method will be something like,

~~~markdown
```table
---
caption: Caption of table[^note2]
markdown: true
---
Col1, Col2
A," B

[^note2]: Description for table."
```
~~~

which take advantage of the fact in pantable v0.13+ all the markdown cells are processed in a batch, allowing them to see the footnote anchor in other locations. So you could put all your footnotes in the last cell in your table and expect them to work.

Actually the creation of this example exposes a weakness and fixed in commit 8535aac4af2ad07d9dd1f06401be5065afb86f01, which leads to v0.13.1.
